driver-embedded-android/src/androidTest/java/com/mongodb/embedded/client/MongoClientsTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.embedded.client;
+
+import android.support.test.runner.AndroidJUnit4;
+
+import com.mongodb.client.MongoClient;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static com.mongodb.embedded.client.Fixture.getMongoClientSettings;
+import static com.mongodb.embedded.client.Fixture.getMongoEmbeddedSettings;
+import static org.junit.Assert.fail;
+
+@RunWith(AndroidJUnit4.class)
```

Comment:
This annotation is for forward compatibility reasons as the default may update to junit 5.

Reply:
Acknowledged